docs/playwright.md

```diff
@@ -83,6 +74,47 @@ module.exports = {
 };
 ```
 
+### With Playwright fixtures
+
+If you're using
+[Playwright fixtures](https://playwright.dev/docs/test-fixtures), you can mix
+happo-playwright in with
+[`mergeTests`](https://playwright.dev/docs/test-fixtures#combine-custom-fixtures-from-multiple-modules).
+
+Here's an example of how to do this:
+
+```js
+// tests/test.js
+import { test as happoTest } from 'happo-playwright';
+import { test as base, mergeTests } from '@playwright/test';
+
+const baseTest = base.extend({
+  myFixture: async ({}, use) => {
+    await use('my fixture value');
+  },
+});
+
+export const test = mergeTests(baseTest, happoTest);
+```
+
+Then in your Playwright test file, you can use the `test` function as usual:
+
+```js
+// tests/test.spec.js
+import { test } from './test';
+
+test('start page', async ({ page, happoScreenshot }) => {
```

Comment:
Maybe add `myFixture` here as well to highlight that you get access to them all.

Reply:
I tried but it felt a little silly. This is fine.